### PR TITLE
Small bugfixes

### PR DESCRIPTION
**Pull request by chyku**
## Description :star2:

Fixes:
- mentor gallery links work now
- remove settings tab
- remove notif bell
- updated home text
- remove home link on mentor sidebar

## Screenshots
image
image

### CODE REVIEW

File: frontend/src/components/MentorCard.js [hunk 1]
```diff
@@ -74,15 +74,23 @@ function MentorCard(props) {
         {props.website && (
           <h4>
             <LinkOutlined style={styles.icon} />
-            <a className="gallery-links" href={props.website}>
+            <a
+              className="gallery-links"
+              href={"//" + props.website}
```
Comment (AngelaLuo49021) [inline frontend/src/components/MentorCard.js:79]: i think you need "://"

File: frontend/src/components/MentorCard.js [hunk 2]
```diff
@@ -74,15 +74,23 @@ function MentorCard(props) {
...
             </a>
           </h4>
         )}
         {props.linkedin && (
           <h4>
             <LinkedinOutlined style={styles.icon} />
-            <a className="gallery-links" href={props.linkedin}>
+            <a
+              className="gallery-links"
+              href={"//" + props.linkedin}
```
Comment (AngelaLuo49021) [inline frontend/src/components/MentorCard.js:91]: same here, it dont work when i tested it

**Review by AngelaLuo49021 — CHANGES_REQUESTED** [review]
Also on the mentor's profiles both in the portal and on the page, you can't click the link to access it. 
image

File: frontend/src/components/MentorCard.js (hunk 1 above)
Comment (chyku) [reply frontend/src/components/MentorCard.js:79]: this worked tho lol

File: frontend/src/components/MentorCard.js (hunk 2 above)
Comment (chyku) [reply frontend/src/components/MentorCard.js:91]: oh wtf

**Review by Leundai — APPROVED** [review]
lgtm!

**Review by AngelaLuo49021 — APPROVED** [review]
Looks good!

**Pull request merged**